### PR TITLE
fix: fix cannot get user comment and remove user_id on anonymous comment

### PR DESCRIPTION
**Pull request by agit26milan**

### CODE REVIEW

File: utils/custom.js
Comment (fajarism) [inline utils/custom.js:1]: Looks like parameter anonymId in handleAnonymousData function is not used.
Please review the function again or remove anonymId param.

File: utils/getAnonymUser.js
Comment (fajarism) [inline utils/getAnonymUser.js:1]: If code enter catch block please return null instead.
We don't want to accidentally expose user's signed id

File: utils/custom.js
Comment (fajarism) [reply utils/custom.js:1]: on line 106, please add null check (?) so it will be like this
`if(child?.data?.anon_user_info_emoji_name)`

Comment (fajarism) [reply utils/custom.js:1]: Also please add (?) null check on all property accessing.

File: controllers/feeds/reactionList.js
Comment (fajarism) [inline controllers/feeds/reactionList.js:1]: Related to comment in utils/custom.js

Please review again anonymId in this param. If it is not used please remove it.

**Review by fajarism — CHANGES_REQUESTED** [review]

File: utils/custom.js
Comment (agit26milan) [reply utils/custom.js:1]: Parameter anonymId is used in  handleAnonymousData function mas

Comment (fajarism) [reply utils/custom.js:1]: Okay Mas

**Review by fajarism — APPROVED** [review]

**Comment by fajarism** [comment]
Great job mas @agit26milan

**Pull request merged**